packages/survey-creator-react/src/components/tabs/TabBase.tsx

```diff
@@ -0,0 +1,12 @@
+import { Base } from "survey-core";
+import { SurveyElementBase } from "survey-react-ui";
+
+export abstract class TabBaseComponent<P, S> extends SurveyElementBase<P, S> {
+  private cachedState: Base;
```

Comment:
I don't understand why we need to cache state element. It should be created in constructor or passed as a prop. And getting state element is a simple and quick operation

Reply:
@tsv2013 We pass in props it as: plugin.model and we make model undefined on deactivating the tab in plug-in. As result props.data.model is underfined and model continue to send notification into unmouned react Component, it generates a correct warning, the Component could not be disposed.
We can theoretically cached it in constructor, but in this case if we can new props on didUpdate, we will be in trouble as well.

Thank you,
Andrew

Reply:
@tsv2013 I think we need to pass into props the model directly and not plugin.model. It would solve the issue.
Let me try to change it.

Thank you,
Andrew

Reply:
@tsv2013 I have changed the commit and pass the model directly into props and remove plug-ins at all. We do not need them in our components. I think we should make the same change in knockout. I have added it in TODO list.

Thank you,
Andrew